packages/mermaid/src/rendering-util/rendering-elements/nodes.js

```diff
@@ -17,36 +17,222 @@ import { doublecircle } from './shapes/doubleCircle.js';
 import { question } from './shapes/question.js';
 import { hexagon } from './shapes/hexagon.js';
+import { text } from './shapes/text.js';
+import { card } from './shapes/card.js';
+import { shadedProcess } from './shapes/shadedProcess.js';
+import { anchor } from './shapes/anchor.js';
 import { lean_right } from './shapes/leanRight.js';
 import { lean_left } from './shapes/leanLeft.js';
 import { trapezoid } from './shapes/trapezoid.js';
 import { inv_trapezoid } from './shapes/invertedTrapezoid.js';
 import { labelRect } from './shapes/labelRect.js';
+import { triangle } from './shapes/triangle.js';
+import { halfRoundedRectangle } from './shapes/halfRoundedRectangle.js';
+import { curvedTrapezoid } from './shapes/curvedTrapezoid.js';
+import { slopedRect } from './shapes/slopedRect.js';
+import { bowTieRect } from './shapes/bowTieRect.js';
+import { dividedRectangle } from './shapes/dividedRect.js';
+import { crossedCircle } from './shapes/crossedCircle.js';
+import { waveRectangle } from './shapes/waveRectangle.js';
+import { tiltedCylinder } from './shapes/tiltedCylinder.js';
+import { trapezoidalPentagon } from './shapes/trapezoidalPentagon.js';
+import { flippedTriangle } from './shapes/flippedTriangle.js';
+import { hourglass } from './shapes/hourglass.js';
+import { taggedRect } from './shapes/taggedRect.js';
+import { multiRect } from './shapes/multiRect.js';
+import { linedCylinder } from './shapes/linedCylinder.js';
+import { waveEdgedRectangle } from './shapes/waveEdgedRectangle.js';
+import { lightningBolt } from './shapes/lightningBolt.js';
+import { filledCircle } from './shapes/filledCircle.js';
+import { multiWaveEdgedRectangle } from './shapes/multiWaveEdgedRectangle.js';
+import { windowPane } from './shapes/windowPane.js';
+import { linedWaveEdgedRect } from './shapes/linedWaveEdgedRect.js';
+import { taggedWaveEdgedRectangle } from './shapes/taggedWaveEdgedRectangle.js';
+import { curlyBraceLeft } from './shapes/curlyBraceLeft.js';
+import { curlyBraceRight } from './shapes/curlyBraceRight.js';
+import { curlyBraces } from './shapes/curlyBraces.js';
 
+//Use these names as the left side to render shapes.
 const shapes = {
+  // States
   state,
   stateStart,
+  'small-circle': stateStart,
+  'sm-circ': stateStart,
+  start: stateStart,
   stateEnd,
-  fork: forkJoin,
-  join: forkJoin,
-  choice,
-  note,
-  roundedRect,
+  'framed-circle': stateEnd,
+  stop: stateEnd,
+
+  // Rectangles
   rectWithTitle,
+  roundedRect,
+  rounded: roundedRect,
+  event: roundedRect,
   squareRect,
+  rectangle: squareRect,
+  rect: squareRect,
+  process: squareRect,
+  proc: squareRect,
   stadium,
+  pill: stadium,
+  term: stadium,
+  windowPane,
+  'window-pane': windowPane,
+  'win-pane': windowPane,
+  'internal-storage': windowPane,
+  dividedRectangle,
+  'divided-rectangle': dividedRectangle,
+  'div-rect': dividedRectangle,
+  'div-proc': dividedRectangle,
+  taggedRect,
+  'tagged-rect': taggedRect,
```

Comment:
Won't adding both camelCased and kebab-cased names create confusion?
We should stick with one uniformly.

Reply:
@sidharthv96 I agree we should stick with one, and use it on the kebab-cased (the one with a hyphen between).
@saurabhg772244 @omkarht  Please make these adjustments to be consistent.